experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -142,3 +142,13 @@ function make_lsm_domain(
         surface_domain,
     )
 end
+
+# required by Interfacer
```

Comment:
This will be done in the next PR, which will focus on revamping the `push_pull.jl` file (it wasn't necessary for the `combine_surfaces!` and `update_surface_fractions!` functions here)